pkg/gen/input/workflows/internal/file/create_release_pr.yaml.template

```diff
@@ -92,7 +92,7 @@ jobs:
             version_major=$(echo "${version}" | cut -d "." -f 1)
             version_minor=$(echo "${version}" | cut -d "." -f 2)
             version_patch=$(echo "${version}" | cut -d "." -f 3)
-            if [[ $version_minor = 0 && $version_patch = 0 ]]; then
+            if [[ $version_minor = 0 && $version_patch =~ ^0.* ]]; then
```

Comment:
technically this will match also `X.0.01`
Is this acceptable?

Reply:
Maybe we could use `^0-?.*`.
Or maybe can we simply compare the major version with the currently released major version? Do we have that info?

Reply:
> technically this will match also `X.0.01` Is this acceptable?


I don't think that's a valid semver version. According to [the specification](https://semver.org/#spec-item-2)


> A normal version number MUST take the form X.Y.Z where X, Y, and Z are non-negative integers, and MUST NOT contain leading zeroes

Reply:
fair enough, even though I think we don't validate that in any way